library/block/block_extensions/template.accesswidener

```diff
@@ -0,0 +1,6 @@
+accessWidener v2 named
```

Comment:
could we have a README.txt or something in this module explaining how to add things to the generated access widener?

Reply:
Added!